packages/core/src/features/st-import.ts

```diff
@@ -275,10 +276,16 @@ function validateImports(context: FeatureTransformContext) {
                     (decl) => decl.type === 'decl' && decl.prop === PseudoImportDecl.FROM
                 );
 
-            context.diagnostics.report(diagnostics.UNKNOWN_IMPORTED_FILE(importObj.request), {
-                node: fromDecl || importObj.rule,
-                word: importObj.request,
-            });
+            context.diagnostics.report(
+                diagnostics.UNKNOWN_IMPORTED_FILE(
+                    importObj.request,
+                    context.resolver.getErrorText(importObj)
```

Comment:
wouldn't it be better to call the module once? If you don't want to change `resolveImported`, then maybe add a new method to the resolver that returns a `[result, error]` ?

It would be nice to have something like `getModule` public without in order to get a JS module or stylable meta from import without having to pass symbol name.

Reply:
If make `getModule` public we could. from what I hear that's what you want?

Reply:
it will improve the surrounding of this function that uses `""` for the symbol name

Reply:
is there an issue with making it public?